bromelia/avps/etsi_3gpp/ts_129_272.py

```diff
@@ -649,6 +649,23 @@ def __init__(self, data):
         DiameterAVP.set_vendor_id_bit(self, True)
         Unsigned32Type.__init__(self, data=data, vendor_id=VENDOR_ID_3GPP)
 
+class IdrFlagsAVP(DiameterAVP, Unsigned32Type):
+    """Implementation of IDR-Flags AVP in Section 7.3.103 of 
+    ETSI TS 129 272 V15.08.0 (2019-07).
```

Comment:
Please update the spec to "ETSI TS 129 272 V15.8.0 (2019-07)." (update from 15.08.0 to 15.8.0)